### PR TITLE
[develop] Base Project

### LINKED ISSUE #1: Base project
Create a base project, considerations:
- Standard Gradle project for Java 11
- Base package: org.fundacionjala.pivotal
- Following packages should be created for `main` sources: config, utils.
- Log4j2.properties should be added to `main` resources customizing config parameters.
- Following packages should be created for `test` sources: context, hooks, runner, stepdefs, utils.
- Runner class should be created.
- Following folders should be added to `test` resources: features, schemas.
- Add cucumber.properties file to `test` resources folder.
- Implement build.gradle script based on file-conversion project (customize gradle tasks and verify that they are working as expected).
- Add .gitignore to root project folder.
- Add gradle.properties file with the following information:
baseUrl=
token=
schemasPath=src/test/resources/schemas/
filterTags=
cucumberThreadCount=1

### PR DESCRIPTION
**Pull request by MirkoFer-622**

### CODE REVIEW

File: src/test/java/org/fundacionjala/pivotal/runner/Runner.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * Cucumber TestNG runner class.
+ */
+@CucumberOptions(
+        plugin = {"pretty"},
+        features = {"src/test/resources/features"},
+        glue = {"org.fundacionjala.pivotal"}
+)
+public final class Runner extends AbstractTestNGCucumberTests {
+    @Override
+    @DataProvider
```
Comment (marcos-x86) [inline src/test/java/org/fundacionjala/pivotal/runner/Runner.java]: Please consider modifying the dataprovider parallel attribute:
```java
@DataProvider(parallel = true)
```

File: src/test/java/org/fundacionjala/pivotal/runner/Runner.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public Object[][] scenarios() {
+        return super.scenarios();
+    }
+}
```
Comment (marcos-x86) [inline src/test/java/org/fundacionjala/pivotal/runner/Runner.java:30]: For now please add the BeforeTest hook to set threadcount (provide the value of 1 as default for now):
```java
    @BeforeTest
    public void beforeAllScenarios() {
        System.setProperty("dataproviderthreadcount", "1");
    }
```

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: src/test/java/org/fundacionjala/pivotal/runner/Runner.java (hunk 1 above)
Comment (MirkoFer-622) [reply src/test/java/org/fundacionjala/pivotal/runner/Runner.java]: I add it at last commit, could you review please?

File: src/test/java/org/fundacionjala/pivotal/runner/Runner.java (hunk 2 above)
Comment (MirkoFer-622) [reply src/test/java/org/fundacionjala/pivotal/runner/Runner.java:30]: It was added at last commit, thanks

**Review by roht0515 — APPROVED** [review]

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        xml.enabled = true
+        html.enabled = true
+    }
+}
+
+check.dependsOn jacocoTestReport
+
+test {
+    useTestNG()
+    exclude 'org/fundacionjala/fc/runner/**'
```
Comment (marcos-x86) [inline build.gradle]: Please consider replacing `org/fundacionjala/fc/runner/**` with `org/fundacionjala/pivotal/runner/**`. Same for the nex line.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

**Review by marcos-x86 — APPROVED** [review]
LGTM!

**Pull request merged**